src/esp/bindings/SensorBindings.cpp

```diff
@@ -112,7 +112,26 @@ void initSensorBindings(py::module& m) {
              VisualSensor, Magnum::SceneGraph::PyFeatureHolder<CameraSensor>>(
       m, "CameraSensor")
       .def(py::init_alias<std::reference_wrapper<scene::SceneNode>,
-                          const SensorSpec::ptr&>());
+                          const SensorSpec::ptr&>())
+      .def("set_projection_params", &CameraSensor::setProjectionParameters,
+           R"(Specify the projection parameters this CameraSensor should use.
+           Should be consumed by first querying this CameraSensor's SensorSpec
+           and then modifying as necessary.)",
+           "sensor_spec"_a)
+      .def(
```

Comment:
Why is this set FOV? It should just be a property FOV that can be modified and has a corresponding setter/getter.